package.json

```diff
@@ -6,22 +6,22 @@
 	"homepage": "https://github.com/microsoft/atlas-design",
 	"scripts": {
 		"test": "echo \"Error: no test specified\" && exit 1",
-		"type-check": "npm run build:css && npm run --prefix site type-check",
+		"type-check": "npm run build:css && npm run type-check --workspace@atlas-design/site",
```

Comment:
`--workspace=@microsoft/atlas-css`
`--workspace=@microsoft/atlas-site`

Name comes from each package's package.json file's name property. And there's an `=` between workspace and the name.